client/lib/Hero.ts

```diff
@@ -410,6 +409,45 @@ export default class Hero extends AwaitedEventTarget<{ close: void }> implements
   public [Util.inspect.custom](): any {
     return inspectInstanceProperties(this, propertyKeys as any);
   }
+
+  // CLASS EVENT EMITTER ///////////////////////////////////////////////////////////////////////////////////////////////
+
+  public static addListener<K extends keyof IClassEvents>(
+    eventType: K,
+    listenerFn: (event: IClassEvents[K]) => any,
+  ): void {
+    this.emitter.addListener(eventType, listenerFn);
+  }
+
+  public static removeListener<K extends keyof IClassEvents>(
+    eventType: K,
+    listenerFn: (event: IClassEvents[K]) => any,
+  ): void {
+    this.emitter.removeListener(eventType, listenerFn);
+  }
+
+  // aliases
+
+  public static on<K extends keyof IClassEvents>(
+    eventType: K,
+    listenerFn: (event: IClassEvents[K]) => any,
+  ): void {
+    return this.addListener(eventType, listenerFn);
+  }
+
+  public static off<K extends keyof IClassEvents>(
+    eventType: K,
+    listenerFn: (event: IClassEvents[K]) => any,
+  ): void {
+    return this.removeListener(eventType, listenerFn);
+  }
+
+  public static once<K extends keyof IClassEvents>(
+    eventType: K,
+    listenerFn: (event: IClassEvents[K]) => any,
+  ): void {
+    return this.addListener(eventType, listenerFn);
```

Comment:
Should this actually implement once?

Reply:
Databox won't implement once, but why shouldn't the functionality be there?

Reply:
It just needs to call this.emitter.once... is my point ;)

Reply:
Ah. Good catch. Pushed fix.